recipes-core/images/u96v2-sbc-dualcam-image.bb

```diff
@@ -14,7 +14,4 @@ IMAGE_INSTALL_ultra96v2 += "\
 		libdrm-kms \
 "
 
-IMAGE_INSTALL_remove_ultra96v2 = "\
-		packagegroup-petalinux-x11 \
-		packagegroup-petalinux-matchbox \
-"
+#IMAGE_INSTALL_remove_u96v2-sbc = "packagegroup-petalinux-x11 packagegroup-petalinux-matchbox"
```

Comment:
Same here. I'm not a huge fan of pushing comments. I like clean commits with an explanation on why we're removing this rather than leaving a comment that will most likely never be uncommented.

Reply:
agreed